wp-content/themes/missilethreat/assets/_scss/style.scss

```diff
@@ -19,7 +21,9 @@
 @use 'components/page-header';
 @use 'components/post-block';
 @use 'components/post-meta';
+@use 'components/post-typography';
```

Comment:
Instead of importing post specific stuff as a part of your `style.scss`, I would import them in either `single.scss` (if needed for both posts & pages) or in `post.scss` or `page.scss` if only needed for one of them.